rplugin/python3/deoplete/sources/tabnine.py

```diff
@@ -66,17 +66,21 @@ def get_complete_position(self, context):
         m = re.search(r'\s+$', context['input'])
         if m:
             return -1
+
         self._response = self._get_response(context)
-        if self._response is None:
+        if self._response is None or 'old_prefix' not in self._response:
             return -1
+
         old_prefix = self._response['old_prefix']
         if not context['input'].endswith(old_prefix):
             return -1
         return len(context['input']) - len(old_prefix)
 
     def gather_candidates(self, context):
-        response = self._response
+        if not self._response:
```

Comment:
Could it be [] here as well? `is None` is preferred if you want to check for None.

Reply:
I have changed it.